src/uu/printenv/src/printenv.rs

```diff
@@ -7,69 +7,67 @@
 
 /* last synced with: printenv (GNU coreutils) 8.13 */
 
-extern crate getopts;
+extern crate clap;
 
 #[macro_use]
 extern crate uucore;
 
+use clap::{App, Arg};
 use std::env;
 
-static NAME: &str = "printenv";
+static ABOUT: &str = "Prints the given environment VARIABLE(s), otherwise prints them all.";
```

Comment:
```suggestion
static ABOUT: &str = "Display the values of the specified environment VARIABLE(s), or (with no VARIABLE) display name and value pairs for them all.";
```

Slightly more precise wording, similar to use elsewhere in `coreutils` and closer to GNU about text.